src/Infrastructure.VS/ThreadHandling.cs

```diff
@@ -19,15 +19,17 @@
  */
 
 using System;
-using System.ComponentModel.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
 using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Threading;
 using SonarLint.VisualStudio.Core;
+using static Microsoft.VisualStudio.Threading.AwaitExtensions;
 using Task = System.Threading.Tasks.Task;
 
 namespace SonarLint.VisualStudio.Infrastructure.VS
 {
-    [Export(typeof(IThreadHandling))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
+    [ExcludeFromCodeCoverage] // Simple wrapper around hard-to-test VS types
```

Comment:
What happened to the MEF attributes?
The class was MEF-imported by CFamily analysis (i.e. VcxRequestFactory)